paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/AbstractFlinkTableFactory.java

```diff
@@ -116,6 +119,12 @@ static Optional<LogStoreTableFactory> createOptionalLogStoreFactory(
             // Use file store continuous reading
             validateFileStoreContinuous(configOptions);
             return Optional.empty();
+        } else if (configOptions.get(SCAN_MODE) == FROM_SNAPSHOT
+                || configOptions.get(SCAN_MODE) == FROM_SNAPSHOT_FULL) {
+            throw new ValidationException(
+                    String.format(
+                            "%s must be null when you use %s for %s",
+                            LOG_SYSTEM.key(), configOptions.get(SCAN_MODE), SCAN_MODE.key()));
```

Comment:
Use message "Log system does not support %s and %s scan mode" ?

Reply:
resolved.